app/app.pro

```diff
@@ -90,9 +93,21 @@ FORMS += \
 
 DEPENDPATH += .
 
-VERSION = 0.5.4
+VERSION = 0.5.4 #FIXME: use build number from git
 DEFINES += APP_VERSION=\\\"$$VERSION\\\"
 
+win32 {
+    GIT_EXISTS='"\\\"$$system(git --version > nul)\\\""'
+} else {
+    GIT_EXISTS='"\\\"$$system(git --version > /dev/null)\\\""'
+}
+
+GIT_EXISTS {
+    DEFINES += GIT_CURRENT_SHA1="\\\"$$system(git --git-dir $$_PRO_FILE_PWD_/../.git --work-tree $$_PRO_FILE_PWD_/../ rev-parse HEAD)\\\"" \
```

Comment:
Command could be shortened by using `-C`, see docs at <https://git-scm.com/docs/git#git--Cltpathgt> for example.